docs/deploy_features.md

```diff
@@ -16,8 +16,18 @@ You should typically enable these features before running `azd up`. Once you've
 
 We generally find that most developers are able to get high quality answers using GPT 3.5. However, if you want to try GPT-4, you can do so by following these steps:
 
-* In `infra/main.bicep`, change `chatGptModelName` to 'gpt-4' instead of 'gpt-35-turbo'.
-* You may also need to adjust the capacity above that line depending on how much TPM your account is allowed.
+* Execute the following command inside your terminal:
+
+    ```bash
+    azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT chat4
```

Comment:
Hm, I'm not sure it's a good idea to prescribe the deployment name used by developers, as developers often want to reuse existing deployments or have some naming scheme to their deployments. Can you instead add an additional variable like USE_GPT4, so that developers can set the deployment name as desired? (The default can still be chat4 for gpt-4 deployment)

Reply:
on it